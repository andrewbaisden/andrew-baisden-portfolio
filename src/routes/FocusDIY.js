import React from 'react';
import { Helmet } from 'react-helmet';
import Header from '../components/Header';
import Footer from '../components/Footer';
import PortfolioPageFocusDIY from '../components/PortfolioPageFocusDIY';
import CheckPortfolioPage from '../components/CheckPortfolioPage';
// import PortfolioCarouselFocusDiy from '../components/PortfolioCarouselFocusDiy';

const FocusDIY = () => (
  <div>
    <div>
      <Helmet>
        <meta
          name="description"
          itemProp="description"
          content="Focus DIY were looking for a Front-End Developer to join their team for a few months just to help out with the main website."
        />
        <title>Focus DIY - Portfolio - Full Stack Developer | Andrew Baisden | London</title>
        <link rel="canonical" href="https://andrewbaisden.com/portfolio/focus-diy" />
      </Helmet>
    </div>
    <CheckPortfolioPage />
    <Header />
    <PortfolioPageFocusDIY companyBG="#1D4C9A" />
    {/* <PortfolioCarouselFocusDiy /> */}
    <Footer />
  </div>
);

export default FocusDIY;
