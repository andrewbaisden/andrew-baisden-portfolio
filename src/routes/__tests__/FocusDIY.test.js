import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import FocusDIY from '../FocusDIY';

Enzyme.configure({ adapter: new Adapter() });

describe('<FocusDIY />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<FocusDIY />);
    expect(wrapper).toMatchSnapshot();
  });
});
