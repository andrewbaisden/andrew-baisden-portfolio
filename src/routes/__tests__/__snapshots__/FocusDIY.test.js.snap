// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FocusDIY /> renders and matches the snapshot 1`] = `ShallowWrapper {}`;
